packages/core/src/Stepper.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Flex from './Flex'
+import Step from './Step'
+
+const StyledFlex = styled(Flex)`
+  & > :not(:last-child) {
+    margin-right: 32px;
```

Comment:
perhaps get the spacing from the inherited theme provider `props => props.theme.spacing[x]`